db/migrate/20240315091658_change_claims_eligibility_type_to_policies_levelling_up_premium_payments.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeClaimsEligibilityTypeToPoliciesLevellingUpPremiumPayments < ActiveRecord::Migration[7.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        Claim.where(eligibility_type: "LevellingUpPremiumPayments::Eligibility").update_all(eligibility_type: "Policies::LevellingUpPremiumPayments::Eligibility")
+      end
```

Comment:
You could add this:

```ruby
entity_tag = Time.now.strftime('%Y%m%d%H%M%S')
DfE::Analytics::LoadEntities.new(entity_name: "claims").run(entity_tag:)
```
So that we don't have to re-import manually (same on the `down` method).

Reply:
Used `DfE::Analytics::LoadEntities.new(entity_name: "claims").run` instead as the gem's tag that we use does not support arguments [v1.11.6](https://github.com/DFE-Digital/dfe-analytics/blob/v1.11.6/lib/dfe/analytics/load_entities.rb#L13)